### PR TITLE
Customizable URL for the application

### PR DESCRIPTION
**Pull request by luisfelipefurlan**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
feature

* **What is the current behavior?** (You can also link to an open issue here)
Application only responds to the address /

* **What is the new behavior (if this is a feature change)?**
The application address can be changed at the time of image generation

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
dojot/dojot#1721

* **Other information**:
There are two parameters available in the dockerfile:
BASE_URL: URL used for API calls
APPLICATION_URL: URL used by the application

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,8 +3,13 @@ FROM node:9.11.2-alpine AS basis
 RUN mkdir /data
 WORKDIR /data
 
-ARG DOJOT_VERSION=undefined
+ARG DOJOT_VERSION
```
Comment (mprevide) [inline Dockerfile:6]: Put a comment that explains each argument
And add that in readme

File: src/js/actions/LoginActions.js [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
 import { AbilityUtil } from 'Components/permissions/ability';
+import { appURL } from 'Src/config';
 
 const alt = require('../alt');
 
-const { t } = i18next;
-
 class LoginActions {
     authenticate(login) {
+        console.log(appURL);
```
Comment (mprevide) [inline src/js/actions/LoginActions.js:12]: Remove console.log

File: src/js/containers/full/Full.js [hunk 3]
```diff
@@ -102,13 +102,13 @@ class RightSideBar extends Component {
...
     render() {
-        if (this.props.user == undefined) {
+        if (this.props.user === undefined) {
             console.error('no active user session');
             return null;
         }
 
 
-        const gravatar = `https://www.gravatar.com/avatar/${btoa(this.props.user.username)}?d=identicon`;
+        // const gravatar = `https://www.gravatar.com/avatar/${btoa(this.props.user.username)}?d=identicon`;
```
Comment (mprevide) [inline src/js/containers/full/Full.js:111]: Remove this unused line

File: Dockerfile (hunk 1 above)
Comment (luisfelipefurlan) [reply Dockerfile:7]: ok

File: src/js/actions/LoginActions.js (hunk 2 above)
Comment (luisfelipefurlan) [reply src/js/actions/LoginActions.js]: done

File: src/js/containers/full/Full.js (hunk 3 above)
Comment (luisfelipefurlan) [reply src/js/containers/full/Full.js]: done

**Review by mprevide — APPROVED** [review]
Great job! :)

**Review by wellitonSouza — APPROVED** [review]

**Pull request merged**